runtime/ai/ai.go

```diff
@@ -339,10 +339,20 @@ type Message struct {
 	ContentType MessageContentType `json:"content_type" yaml:"content_type"`
 	// Content is the content of the message.
 	Content string `json:"content" yaml:"content"`
+	// Context passed to the message that is used by different tool calls
+	Context MessageContext `json:"context" yaml:"context"`
```

Comment:
Everything in the `AnalystAgentArgs` is already serialized and stored in `Content`, so this is redundant (and as per the earlier message, agent-specific properties shouldn't go in the `ai.go` file)

Reply:
This is a holdover of some older code. Not used anymore, removing it.